### PR TITLE
Emits a details event when FAST top-level parameters change.

### PR DESCRIPTION
**Pull request by hickscorp**

### CODE REVIEW

File: contracts/fast/FastFrontendFacet.sol [hunk 1]
```diff
@@ -38,12 +52,28 @@ contract FastFrontendFacet is AFastFacet {
...
+    LibFastAccess.Data storage accessData = LibFastAccess.data();
+    LibFastToken.Data storage tokenData = LibFastToken.data();
+    emit DetailsChanged({
+      memberCount: accessData.memberSet.values.length,
+      governorCount: accessData.governorSet.values.length,
+      totalSupply: tokenData.totalSupply,
+      transferCredits: tokenData.transferCredits,
+      reserveBalance: tokenData.balances[LibConstants.ZERO_ADDRESS],
+      ethBalance: payable(address(this)).balance
+    });
```
Comment (jamesduncombe) [inline contracts/fast/FastFrontendFacet.sol:68]: Are we going to need to keep expanding this as and when use cases come up?

File: contracts/fast/FastTopFacet.sol [hunk 2]
```diff
@@ -52,5 +54,6 @@ contract FastTopFacet is AFastFacet {
     amount = LibHelpers.upTo(recipient, amount);
     // Transfer some eth!
     if (amount != 0) { recipient.transfer(amount); }
```
Comment (jamesduncombe) [inline contracts/fast/FastTopFacet.sol:56]: Wouldn't this call back into `transfer` which in turn would fire a `DetailsChanged` type event?

File: deploy/11_fundSpc.ts [hunk 3]
```diff
@@ -13,6 +13,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const spc = await ethers.getContract('Spc') as Spc;
   console.log(`Funding the SPC at ${spc.address} with 10_000 ETH...`);
   spc.connect(storageSigner).provisionWithEth({ value: toBaseUnit(10_000, 18) });
+
+  await wait(8000)
```
Comment (jamesduncombe) [inline deploy/11_fundSpc.ts:17]: Why do we `wait` ? :)

Comment (hickscorp) [reply deploy/11_fundSpc.ts:17]: Because if we queue all the transactions the gas fee increase and the deployments fail in the middle.

Comment (jamesduncombe) [reply deploy/11_fundSpc.ts:17]: Ah ok!

**Pull request merged**